core/src/main/scala/filodb.core/memstore/TimeSeriesShard.scala

```diff
@@ -905,9 +903,9 @@ class TimeSeriesShard(val ref: DatasetRef,
         dirtyParts += p.partID
         activelyIngesting -= p.partID
```

Comment:
Can we check the size of ``activelyIngesting`` before and after removal and ensure the different is 1 before we decrement the count. Just trying to eliminate the fact that we don't get duplicate ``TimeSeriesPartition`` objects for whatever reason.

Reply:
log added